pkg/docker/docker.go

```diff
@@ -609,15 +610,30 @@ func (d *stiDocker) RunContainer(opts RunContainerOptions) error {
 
 	containerName := containerNameOrID(container)
 
-	// Container was created, so we defer its removal.
-	defer func() {
+	// Container was created, so we defer its removal, and also remove it if we get a sigterm
```

Comment:
s/sigterm/SIGTERM?

I'd keep the full stop at the end of the comment for consistency.